README.md

```diff
@@ -109,7 +109,7 @@ This project uses the `project.json` for the project build system. Please see th
 
 To build the Open XML SDK:
 - Clone the repo at https://github.com/OfficeDev/Open-XML-SDK
-- Open the solution using Visual Studio 2015.  Community Edition of VS2015 works just fine.
+- Open the solution using Visual Studio 2017.  Community Edition of VS2017 works just fine.
```

Comment:
would this link be right: [https://msdn.microsoft.com/en-us/library/ms246609.aspx](url)?